packages/deployment/app/cloudformation.template.yml

```diff
@@ -655,7 +685,11 @@ Resources:
                 {{else}}
                   - Example: value
                 {{/if}}
-                  
+          
+          packages:
+            yum:
+              nfs-utils: [] 
```

Comment:
this should not be required if there is no efs service mounted

Reply:
ok